### PR TITLE
fix press-cards image download

### PR DESCRIPTION
**Pull request by alinarublea**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/ausgezeichnete-ausbildung-bei-zeiss
- After: https://issue-fix-image-download--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/ausgezeichnete-ausbildung-bei-zeiss

### CODE REVIEW

File: blocks/press-cards/press-cards.js [hunk 1]
```diff
@@ -76,6 +76,7 @@ export default async function decorate(block) {
     });
     ul.append(li);
   });
+  ul.querySelectorAll('img').forEach((img) => img.closest('picture').replaceWith(createOptimizedPicture(img.src, img.alt, false, [{ width: '750' }])));
```
Comment (sdmcraft) [inline blocks/press-cards/press-cards.js:79]: What's the issue that we are fixing here?

Comment (alinarublea) [reply blocks/press-cards/press-cards.js:79]: The href is not working for the press cards images

Comment (sdmcraft) [reply blocks/press-cards/press-cards.js:79]: Thanks. But href still does not seem to work for me for [this page](https://issue-fix-image-download--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/ausgezeichnete-ausbildung-bei-zeiss).

Comment (sdmcraft) [reply blocks/press-cards/press-cards.js:82]: > @sdmcraft now it should work

Thanks. One minor change needed though. On the original site, the download happens in a new tab while in our case, it happens in the same tab.

**Comment by sdmcraft** [comment]
> @sdmcraft now it should work

Thanks. One minor change needed though. On the original site, the download happens in a new tab while in our case, it happens in the same tab.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**